UIkitDZ/VC/PaymentVC.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  PaymentVC.swift
+//  UIkitDZ
+//
+//  Created by Timofey Privalov on 03.07.2021.
+//
+
+import UIKit
+
+class PaymentVC: UIViewController {
+    
+    var nameOfPizza = String()
+    var cheese = false
+    var bacon = false
+    var mashrooms = false
+    var olive = false
+    
+    let cardPaySwitch = UISwitch()
+    let cashPaySwitch = UISwitch()
+    
+    var paymentAlert = UIAlertController()
+   
+    weak var delegate: getToLoggedVC?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view.backgroundColor = .white
+        printLabels(foodArr: getOrderItem())
+        createPayButton()
+        
+        cardPaySwitch.frame = CGRect(x: 330, y: 670, width: 20, height: 10)
+        cardPaySwitch.addTarget(self, action: #selector(cashSwitcher), for: .valueChanged)
+        view.addSubview(cardPaySwitch)
+        
+        cashPaySwitch.frame = CGRect(x: 330, y: 720, width: 20, height: 10)
+        cashPaySwitch.addTarget(self, action: #selector(cardSwitcher), for: .valueChanged)
+        view.addSubview(cashPaySwitch)
+        
+        createCashCardLabel(nameType: "Оплата наличкой", x: 155, y: 655)
+        createCashCardLabel(nameType: "Оплата карточкой", x: 155, y: 705)
+
+    }
+    
+    // MARK: создание алерта по клику на кнопку 🍎
+    
+    @objc func createAlert(alertController: UIAlertController) {
+        let alertController = UIAlertController(title: "Заказ оплачен", message: "Ваш заказ оплачен, будет доставлен в течении 15 минут", preferredStyle: .alert)
+        
+        let action = UIAlertAction(title: "Ok", style: .default) { (action) in
+            self.dismiss(animated: false, completion: nil)
+            self.delegate?.getToLogged()
+        }
+        alertController.addAction(action)
+        self.present(alertController, animated: true, completion: nil)
+    }
+    
+    func getToFoodVC() {
+        print("Hi")
+}
+    
+    // MARK: создание функций включения выключения switch
+    @objc private func cashSwitcher(sw: UISwitch) {
+        switch sw.isOn {
+        case true:
+            cashPaySwitch.isOn = false
+        default:
+            break
+        }
+    }
+    
+    @objc private func cardSwitcher(sw: UISwitch) {
+        switch sw.isOn {
+        case true:
+            cardPaySwitch.isOn = false
+        default:
+            break
+        }
+    }
+
+    
+    // MARK: создание лэйблов
```

Comment:
```suggestion
    // MARK: создание лэйблов
```
приведи везде марки в порядок и по гайду